mvpa2/support/nibabel/surf.py

```diff
@@ -412,29 +391,29 @@ def dijkstra_shortest_path(self, src, maxdistance=None):
         nbrs = self.neighbors
 
         # algorithm from wikipedia
-        #(http://en.wikipedia.org/wiki/Dijkstra's_algorithm)
+        # (http://en.wikipedia.org/wiki/Dijkstra's_algorithm)
         while candidates:
             # distance and index of current candidate
             d, i = heapq.heappop(candidates)
 
             if i in fdist:
-                continue # we already have a final distance for this node
+                continue  # we already have a final distance for this node
 
-            nbr = nbrs[i] # neighbours of current candidate
+            nbr = nbrs[i]  # neighbours of current candidate
 
             for nbr_i, nbr_d in nbr.items():
                 dnew = d + nbr_d
 
-                if maxdistance is not None and dnew > maxdistance:
-                    continue # skip if too far away
+                if not maxdistance is None and dnew > maxdistance:
```

Comment:
previous pattern was correct -- "is not" is a preferable way to go.  @hanke even did a full blown change everything PR for that